components/email/src/Email.svelte

```diff
@@ -421,22 +421,9 @@
         }
       }
 
-      //#region open thread + messages
       if (messageType !== MessageType.DRAFTS) {
         activeThread.expanded = !activeThread.expanded;
-        // Upon expansion / lastMessage existing, scroll to it
-        if (activeThread.expanded && _this.click_action === "default") {
-          // Timeout here is to ensure the element is available before trying
-          // to scroll it into view
-          setTimeout(() => {
-            messageRefs[lastMsgIndex].scrollIntoView({
-              behavior: "smooth",
-              block: "start",
-            });
-          }, 50);
-        }
       }
-      //#endregion open thread + messages
```

Comment:
Originally removed [here](https://github.com/nylas/components/commit/9871ca24b2c0e9e60d6ed74b6f092af788ef6e1b#diff-1862c741f2c1c0aed3ed5b79632759d4b6c13ec8b0d81e4508d71fc120582c9eL424), but I left this in by accident in a previous PR

Reply:
Do we not need `scrollIntoView` anymore? I thought we had a story to add scroll in to view?

Reply:
It's been moved to a reactive block [here](https://github.com/nylas/components/blob/d6116512686e8dc2f7db114fa2a17f2292b498dd/components/email/src/Email.svelte#L864-L873)